routers/repo/pull.go

```diff
@@ -237,7 +237,9 @@ func ForkPost(ctx *context.Context, form auth.CreateRepoForm) {
 		}
 		return
 	}
-
+	
+	time.Sleep(1500*time.Millisecond) //wait language stat indexer finish
```

Comment:
I don't think it is good idea

Reply:
> It would be better and faster just to copy stats from original repo

Thanks your suggestion, I will try another way.
Because i am not familiar with the repo and language stat structure, i use the most easy method.😂
May you give me some tip why the new repo and the original one's size is a little bit different. I worry that directly copy the language stat will cause errors by these difference.

Reply:
This sleep function is not only wait indexer but also database operation, it waits all kinds of asynchronous language statistic operation.
Only if we use a synchronous  language statistic operation, the sleep can be removed.

Reply:
A synchronous language statistic operation means all the language statistic information had written in database. Then execute next statement. When operating database this goroutine(func ForkPost) should be blocked.